apps/prairielearn/assets/scripts/navbarClient.ts

```diff
@@ -8,6 +11,23 @@ onDocumentReady(() => {
   // The navbar is not present in some pages (e.g., workspace pages), in that case we do nothing.
   if (!usernameNav) return;
 
+  // Ideally we'd have HTMX listen for the `show.bs.dropdown` event, but
+  // Bootstrap 4 doesn't use native browser events. Once we upgrade to
+  // Bootstrap 5, we can update the HTMX trigger to use the native event.
```

Comment:
🤮 there are probably other ways we could do this. Maybe it would make more sense to use the same event name (`show.bs.dropdown`) and emit that directly on the `#navbar-...-switcher` elements, like Bootstrap 5 would?

Reply:
Doesn't seem to really matter either way? It's a hack until we upgrade to BS5 in any case.